models/block.js

```diff
@@ -0,0 +1,16 @@
+var mongoose = require('mongoose');
```

Comment:
Change name of file (blockSchema or whatever) and use block model from node module in the schema